### ISSUE TITLE
Kubernetes Community-Owned Package Repositories at pkgs.k8s.io missing cri-tools 1.27 debian package

### ISSUE THREAD

**Issue by haiwu** [kubernetes/release#3218]
#### What happened:

Kubernetes Community-Owned Package Repositories at pkgs.k8s.io missing cri-tools 1.27 debian package

I followed the steps listed here: https://kubernetes.io/blog/2023/08/15/pkgs-k8s-io-introduction/, and used these: 
```
echo "deb [signed-by=/etc/apt/keyrings/kubernetes-apt-keyring.gpg] https://pkgs.k8s.io/core:/stable:/v1.28/deb/ /" | sudo tee /etc/apt/sources.list.d/kubernetes.list
echo "deb [signed-by=/etc/apt/keyrings/kubernetes-apt-keyring.gpg] https://pkgs.k8s.io/core:/stable:/v1.27/deb/ /" | sudo tee -a /etc/apt/sources.list.d/kubernetes.list
```

and I could only find cri-tools 1.25, 1.26, 1.28 for Debian. But cri-tools 1.27 is nowhere to be found. 

#### What you expected to happen:
There should be cri-tools debian package for 1.27, now that cri-tools 1.28 package is already out. 

#### How to reproduce it (as minimally and precisely as possible):
Just followed the steps listed here: https://kubernetes.io/blog/2023/08/15/pkgs-k8s-io-introduction/, and used these: 
```
echo "deb [signed-by=/etc/apt/keyrings/kubernetes-apt-keyring.gpg] https://pkgs.k8s.io/core:/stable:/v1.28/deb/ /" | sudo tee /etc/apt/sources.list.d/kubernetes.list
echo "deb [signed-by=/etc/apt/keyrings/kubernetes-apt-keyring.gpg] https://pkgs.k8s.io/core:/stable:/v1.27/deb/ /" | sudo tee -a /etc/apt/sources.list.d/kubernetes.list
```

#### Anything else we need to know?:
N/A

**Comment by saschagrunert** [comment]
@xmudrii we probably need to backfill cri-tools v1.27. Are you fine with that?

**Comment by xmudrii** [comment]
Thank you for bringing this up, @haiwu! cri-tools v1.27.1 is now available in project for v1.27:

```
 cri-tools | 1.27.1-1.1 | https://pkgs.k8s.io/core:/stable:/v1.27/deb  Packages
 cri-tools | 1.26.0-1.1 | https://pkgs.k8s.io/core:/stable:/v1.27/deb  Packages
 cri-tools | 1.25.0-1.1 | https://pkgs.k8s.io/core:/stable:/v1.27/deb  Packages
```
